sdks/python/apache_beam/ml/inference/pytorch_inference.py

```diff
@@ -170,6 +183,9 @@ def run_inference(
 
     For the same key across all examples, this will stack all Tensors values
     in a vectorized format to optimize the inference call.
+
```

Comment:
Can we add inputs here?

Reply:
Done.  @yeandy I've made these inputs fairly light. I was able to make the sklearn inputs more specific. Is there any specificity you recommend for these parameters?